src/PostProcessor.cpp

```diff
@@ -405,10 +427,21 @@ void PostProcessor::_initBlur()
 	glUseProgram(0);
 }
 
+void PostProcessor::_initOrientationCorrection()
+{
+	m_orientationCorrectionProgram = _createShaderProgram(vertexShader, orientationCorrectionShader);
+	glUseProgram(m_orientationCorrectionProgram);
+	int loc = glGetUniformLocation(m_orientationCorrectionProgram, "Sample0");
+	assert(loc >= 0);
+	glUniform1i(loc, 0);
+	glUseProgram(0);
+}
+
 void PostProcessor::init()
 {
 	_initCommon();
 	_initGammaCorrection();
+	_initOrientationCorrection();
```

Comment:
let's make _initOrientationCorrection call optional, as for bloom.
why create resources, which will not be used?